### ISSUE TITLE
Different folder structure results in Windows vs Linux: missing src folder

### ISSUE THREAD

**Issue by ChrisBrownie55** [benawad/destiny#142]
When running the command `yarn destiny ./src` the results are different between Windows and Linux. Linux behaves as expected and organizes the files and directories inside of the `src` directory, whereas, Windows places the files at the root of the project.

### My project file structure:

```tree
.
├── package.json
├── src
│   ├── App.css
│   ├── App.jsx
│   ├── index.html
│   ├── logo.svg
│   ├── main.jsx
│   └── styles.css
├── vite.config.js
└── yarn.lock
```

### Linux result (works as expected):
```tree
INFO: Generating tree for: ./src
src
├──main
│  ├──App
│  │  ├──App.css
│  │  └──logo.svg
│  ├──App.jsx
│  └──styles.css
└──main.jsx
```

### Windows result (very much not expected):
```tree
INFO: Generating tree for: ./src

├──main.jsx
├──main\App.jsx
├──main\App\App.css
├──main\App\logo.svg
└──main\styles.css
```

I think it for some reason just replaced `src` with nothing because the file structure is correct but it's simply not in the right place, however, it also displays weird as I wouldn't have expected the output of a tree to be flat.

Left: Ubuntu Subsystem
Right: Windows 10
![Screenshots](https://user-images.githubusercontent.com/19195374/82620852-291b9400-9b9f-11ea-9a47-d3196d7be123.png)

**Comment by AnatoleLucet** [comment]
Destiny is moving every files out of `src` for the same reason as #135.

It seems like there's also an issue with the way that Destiny print the tree. I've tried to run `destiny src -w` using the build from #140  on a Vite React template and it correctly prettify the file structure, but the printed tree is the same as on your screenshot (except all the files were in the `src` folder).

**Comment by ChrisBrownie55** [comment]
@AnatoleLucet Yeah I just tried out that PR too (had to fork it to change npm build script to work when doing `yarn add` with a GitHub repo) and it fixed the src thing but yeah, is the tree generating thing just not recognizing the `\` as a valid separator or something?

**Comment by AnatoleLucet** [comment]
> is the tree generating thing just not recognizing the \ as a valid separator or something?

Yep, this might be it.
